content/projects/MORPHEUS.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Morpheus
+date: 2023-08-25
+publishdate: 2023-08-25
+weight: 20
+nav: false
+people:
+- key: floriane.odje
+- key: andrea.volkamer
+funding:
```

Comment:
also add the collaborators category and add the people from BfR and FMP, including links.